### PR TITLE
[WIP] Enable no-implicit-any

### PR DESCRIPTION
**Pull request by HenryNguyen5**

### CODE REVIEW

File: common/components/BalanceSidebar/TokenBalances/Balances.tsx [hunk 1]
```diff
@@ -15,19 +15,23 @@ interface Props {
   onRemoveCustomToken(symbol: string): any;
 }
 
+interface TrackedTokens {
+  [symbol: string]: boolean;
+}
+
 interface State {
-  trackedTokens: { [symbol: string]: boolean };
+  trackedTokens: any;
```
Comment (dternyak) [inline common/components/BalanceSidebar/TokenBalances/Balances.tsx]: not `TrackedTokens`?

File: common/components/BalanceSidebar/TokenBalances/Balances.tsx [hunk 2]
```diff
@@ -37,7 +41,8 @@ export default class TokenBalances extends React.PureComponent<Props, State> {
 
   public render() {
     const { allTokens, tokenBalances, hasSavedWalletTokens } = this.props;
-    const { showCustomTokenForm, trackedTokens } = this.state;
+    const { showCustomTokenForm } = this.state;
+    const trackedTokens: any = this.state.trackedTokens;
```
Comment (dternyak) [inline common/components/BalanceSidebar/TokenBalances/Balances.tsx]: `TrackedTokens`, not `any`

File: common/components/BalanceSidebar/TokenBalances/Balances.tsx [hunk 3]
```diff
@@ -129,7 +134,7 @@ export default class TokenBalances extends React.PureComponent<Props, State> {
   };
 
   private handleSetWalletTokens = () => {
-    const { trackedTokens } = this.state;
+    const trackedTokens: any = this.state.trackedTokens;
```
Comment (dternyak) [inline common/components/BalanceSidebar/TokenBalances/Balances.tsx]: TrackedTokens, not any

File: common/containers/Tabs/Swap/components/CurrentRates.tsx [hunk 4]
```diff
@@ -42,7 +43,7 @@ export default class CurrentRates extends PureComponent<Props> {
 
   public buildSSPairs = (shapeshiftRates: NormalizedShapeshiftRates, n: number = 4) => {
     const pairCollection = times(n, () => this.getRandomSSPairData(shapeshiftRates));
-    const byId = pairCollection.reduce((acc, cur) => {
+    const byId = pairCollection.reduce((acc: any, cur) => {
```
Comment (dternyak) [inline common/containers/Tabs/Swap/components/CurrentRates.tsx]: `cur: any`

**Comment by HenryNguyen5** [comment]
Closing since stale.

**Pull request closed without merging**